README.md

```diff
@@ -71,25 +70,21 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **[Displays List of ToDo Tasks]**
-- **[Addition of New Tasks]**
-- **[Removal of a Task from the list]**
+- **[Displays List of Players and Scores]**
+- **[Addition of New Players and Scores]**
+- **[Removal of all Players and Scores from the list]**
 - **[Local storage]**
-- **[Editing a task with Edit Button]**
-- **[Checkboxes are there to check the completed tasks]**
-- **[All the completed tasks can be Cleared by Clicking the Clear All Completed Button]**
-- **[A Reset button is also there to reset all the tasks.]**
+- **[A Reset button is also there to reset all the Players and Scores.]**
 - **[Webpack is managing all the dependencies and auto loading in browser]**
 - **[All the scetions and functionality is re-created by ES6 Modules]**
-- **[ToDo List is Interactive, checking and unchecking the checkbox is interactive.]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- <a href="https://mike111222.github.io/ToDoList/dist/">My ToDoList Project's Live Demo Link</a>
+- Coming Soon
```

Comment:
- [ ] Kindly delete it completely since it's not implemented yet just as instructed by the last reviewer. You'll add it up later when you deploy the project 👍 . Let us avoid having redundant sections that make the README clustered 👍

Reply:
Thanks a lot for your positive words. I have done the changes you mentioned. Thanks.